crates/libp2p-networking/src/network/node.rs

```diff
@@ -299,13 +300,25 @@ impl NetworkNode {
                 ..Default::default()
             };
 
+            // Limit the number of simultaneous connections
+            let connection_limits = libp2p::connection_limits::Behaviour::new(
+                ConnectionLimits::default()
+                    .with_max_established(Some(100))
+                    .with_max_established_incoming(Some(70)) // To prevent eclipse attacks
+                    .with_max_established_outgoing(Some(100))
+                    .with_max_established_per_peer(Some(1)) // Just reuse the one we already have
```

Comment:
I'm not sure but we might want this to be 2 or more, I don't know what is considered a connection but it seems like 1 incoming and one outgoing connection will be established at least.  Idk the docs don't make it clear

Reply:
Looked through the code I think this needs to be 2 so we can establish and incoming and outgoing connection at the same time.